go/systests/teams_test.go

```diff
@@ -255,6 +255,18 @@ func (u *userPlusDevice) teamSetSettings(teamName string, settings keybase1.Team
 		Settings: settings,
 	})
 	require.NoError(u.tc.T, err)
+	changeByID := false
+	for {
+		select {
+		case arg := <-u.notifications.changeCh:
+			changeByID = arg.Changes.Misc
+		case <-time.After(500 * time.Millisecond * libkb.CITimeMultiplier(u.tc.G)):
```

Comment:
Probably shouldn't do all that in `userPlusDevice.teamSetSettings`, no reason to wait on this notification in every test. A specific test checking if notifications comes after setSettings should be enough.

Unless this is necessary to drain gregor of this notifications because continuing.

Reply:
I thought it would be good to put here to make sure we have full coverage. anytime we update settings without error we should get the notification.